core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/add-hint-modal.component.ts

```diff
@@ -73,7 +73,29 @@ export class AddHintModalComponent
   }
 
   isHintLengthExceeded(tmpHint: string): boolean {
-    return Boolean(tmpHint.length > 500);
+    // The variable chars stores the remaining characters by
+    // removing the html tags from tmpHint.
+    let charRgx = /(?<=>)([^(\n)]*?)(?=<)/g;
+    let chars: string = '';
```

Comment:
"chars" is too generic, can we rename this so that it is clearer what it represents in this context?